JS_1_pow.js

```diff
@@ -0,0 +1,26 @@
+var x = prompt('Введите число x:');
+var n = prompt('Укажите степень n (целое число):');
+
+if (n >= 0) {
+  function powPositiveExp(x, n) {

```

Comment:
Defining named functions inside `if(){}` scope doesn't make them conditionally defined. You will have both in any case. It makes sense to define them outside of `if`

Reply:
done